### ISSUE TITLE
Error building site: EXECUTE-AS-TEMPLATE | Fedora

### ISSUE THREAD

**Issue by FelipeRuizGarcia** [localstack/docs#147]
Unable to run Hugo server in the main branch 
```commit 287fa6f102652328c8fc54f8e0444879e105f150 (HEAD -> main, origin/main, origin/HEAD) ```

After clone and submodules updated, output says:
```
`~/repos/docs │ main  hugo serve                                                                                                    ✔ │ 8m 17s │ 08:53:18 PM 
Start building sites … 

Built in 1286 ms
Error: Error building site: EXECUTE-AS-TEMPLATE: failed to transform "json/offline-search-index.json" (application/json): failed to parse Resource "json/offline-search-index.json" as Template:: template: json/offline-search-index.json:5: unclosed action
 ~/repos/docs │ main 
 `
```

SYSTEM
FEDORA 35

Hugo Static Site Generator v0.80.0/extended linux/amd64 BuildDate: unknown

**Comment by FelipeRuizGarcia** [comment]
Task: test with the recent Hugo version

**Comment by FelipeRuizGarcia** [comment]
Problem Solved:
The root cause was the Hugo Version: The fix is included in the latest Hugo version  `v0.97.3`

Solution: The easiest way is get the latest hugo from brew! 
https://formulae.brew.sh/formula/hugo#default
